src/main/plugin/iso19139.ca.HNAP/loc/eng/schematron-rules-common.xml

```diff
@@ -56,7 +56,6 @@
   <ServiceNamespace>srv:SV_ServiceIdentification is misplaced for gmd:MD_ScopeCode: </ServiceNamespace>
   <MissingContactMail>Contact - Electronic mail address is required</MissingContactMail>
 
-  <requireCitation>Citation</requireCitation>
-  <alert.requiredCitation>Data Identification Citation is a required element</alert.requiredCitation>
+  <hasDuplicatedOnlineResource>Duplicated Online Resource not allowed. The duplicated online resource "language, url" is:</hasDuplicatedOnlineResource>
```

Comment:
There can be one or more duplicate so you should not end with "is:"  Also, it is not clear that it is not the full resource that cannot be duplicates, it is the resource url and language.

So maybe something this may be better.

`Duplicated Online Resource URL's for the same language are not allowed. The following duplicate(s) (language; URL) were identified:`